code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -382,102 +415,79 @@
-		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
-			air.temperature = max(((temperature * old_heat_capacity + energy_used) / new_heat_capacity), TCMB) //the air heats up when reacting
-		return REACTING
-
-/datum/gas_reaction/nitrium_decomposition //The decomposition of nitrium. Exothermic. Requires oxygen as catalyst.
-	priority_group = PRIORITY_PRE_FORMATION
-	name = "Nitrium Decomposition"
-	id = "nitrium_decomp"
-
-/datum/gas_reaction/nitrium_decomposition/init_reqs()
-	requirements = list(
-		/datum/gas/oxygen = MINIMUM_MOLE_COUNT,
-		/datum/gas/nitrium = MINIMUM_MOLE_COUNT,
-		"MAX_TEMP" = T0C + 70 //Pretty warm, explicitly not fire temps. Time bombs are cool, but not that cool. If it makes you feel any better it's close
-	)
-
-/datum/gas_reaction/nitrium_decomposition/react(datum/gas_mixture/air)
-	var/list/cached_gases = air.gases
-	var/temperature = air.temperature
-	var/old_heat_capacity = air.heat_capacity()
-
-	//This reaction is agressively slow. like, a tenth of a mole per fire slow. Keep that in mind
-	var/heat_efficency = min(temperature / (FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 8), cached_gases[/datum/gas/nitrium][MOLES])
-	var/energy_produced = heat_efficency * NITRIUM_DECOMPOSITION_ENERGY
-
-	if ((cached_gases[/datum/gas/nitrium][MOLES] - heat_efficency < 0)) //Shouldn't produce gas from nothing.
-		return NO_REACTION
-
-	air.assert_gases(/datum/gas/nitrogen, /datum/gas/hydrogen)
-	cached_gases[/datum/gas/nitrium][MOLES] -= heat_efficency
-	cached_gases[/datum/gas/hydrogen][MOLES] += heat_efficency
-	cached_gases[/datum/gas/nitrogen][MOLES] += heat_efficency
+	var/energy_used = heat_efficency * N2O_FORMATION_ENERGY
+	var/new_heat_capacity = air.heat_capacity()
+	if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+		air.temperature = max(((air.temperature * old_heat_capacity + energy_used) / new_heat_capacity), TCMB) // The air cools down when reacting.
+	return REACTING
 
-	if(energy_produced> 0)
-		var/new_heat_capacity = air.heat_capacity()
-		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
-			air.temperature = max(((temperature * old_heat_capacity + energy_produced) / new_heat_capacity), TCMB) //the air heats up when reacting
-		return REACTING
 
-/datum/gas_reaction/nitrium_formation //The formation of nitrium. Endothermic. Requires bz.
-	priority_group = PRIORITY_FORMATION
-	name = "Nitrium formation"
-	id = "nitrium_formation"
+/**
+ * Nitrous Oxide Decomposition
+ *
+ * Decomposition of N2O.
+ * Exothermic.
+ */
+/datum/gas_reaction/nitrous_decomp
+	priority_group = PRIORITY_POST_FORMATION
+	name = "Nitrous Oxide Decomposition"
+	id = "nitrous_decomp"
 
-/datum/gas_reaction/nitrium_formation/init_reqs()
+/datum/gas_reaction/nitrous_decomp/init_reqs()
 	requirements = list(
-		/datum/gas/tritium = 20,
-		/datum/gas/nitrogen = 10,
-		/datum/gas/bz = 5,
-		"MIN_TEMP" = 1500
+		/datum/gas/nitrous_oxide = MINIMUM_MOLE_COUNT * 2,
+		"MIN_TEMP" = N2O_DECOMPOSITION_MIN_TEMPERATURE,
+		"MAX_TEMP" = N2O_DECOMPOSITION_MAX_TEMPERATURE,
```

Comment:
Am I blind? Where did this max temp come from?

Reply:
It was implied by the quadratic rate scaling. Temperatures at that or higher made the reaction rate go to zero. I just made it explicit.